lib/active_admin/csv_builder.rb

```diff
@@ -43,7 +43,6 @@ def column(name, options = {}, &block)
     end
 
     def build(controller, csv)
-      @collection = controller.send :find_collection, except: :pagination
```

Comment:
As a note, this maybe a 'private' part of the gem, but this broke an admin view for us. I can't find a way to get at the collection for a CSV download now. Any tips would be appreciated! 

This refactor looks nice, otherwise!

Reply:
Can you show an example of what broke, ideally a public repository, and instructions to reproduce the problem. Thanks!

Reply:
`controller.send :find_collection, except: :pagination`?

Reply:
Great that seems to have worked! I should have tried that first, but was trying to avoid firing another query. 

Thanks so much! 

For posterity: `first_item = controller.send(:find_collection, except: :pagination).first` worked to fix up this code. Thanks!

Reply:
Maybe `first_item = controller.send(:collection).first` works for you to avoid to extra query.

Reply:
I tried that out but ended up switching to `controller.public_send(:find_collection ...` as we override that in the admin class already. Thanks!